CalcCurrentValue/CalcCurrentValue/CalcCurrentValue.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalcCurrentValue
+{
+    class CalcCurrentValue
```

Comment:
Try to give another name for this class. It look like name of method not a class.  CurrentValueCalculator more sutable name for a class